utils/call-gen/src/lib.rs

```diff
@@ -42,6 +42,39 @@ pub struct GearCallConversionError(pub &'static str);
 pub type Seed = u64;
 pub type GearProgGenConfig = gear_wasm_gen::GearConfig;
 
+/// This trait must be implemented for all argument types
+/// that are used in `GearCall::Variant(_)`
+pub trait CallArgs: GeneratableCallArgs + NamedCallArgs {}
+
+impl<T: GeneratableCallArgs + NamedCallArgs> CallArgs for T {}
+
+/// Describes the type that is used for fuzzing by the
+/// `gear-node-load` and `runtime-fuzzer` crates
+pub trait GeneratableCallArgs {
+    /// Describes the arguments of the test environment,
```

Comment:
should be just "arguments"?
```suggestion
    /// Describes arguments of the test environment,
```